packages/scene-composer/src/components/panels/SettingsPanel.tsx

```diff
@@ -92,6 +95,16 @@ export const SettingsPanel: React.FC<SettingsPanelProps> = ({ valueDataBindingPr
           </FormField>
         </SpaceBetween>
       </ExpandableInfoSection>
+
+      {tagResizeEnabled && (
```

Comment:
When are the scenarios where **tagResizeEnabled**  is undefined? What happens if the tagResizeEnabled is undefined?

Reply:
this is our feature flag mechanism to control new feature release. when it's not true, the new settings won't show